javadoc complains about missing @return

Javadoc in JDK 11 complains about missing `@return` annotations for couple functions:
```
[ERROR] foo.java:439: warning: no @return
[ERROR]   public final int yystate() {
[ERROR]                    ^
[ERROR] foo.java:457: warning: no @return
[ERROR]   public final String yytext() {
[ERROR]                       ^
[ERROR] foo.java:481: warning: no @return
[ERROR]   public final int yylength() {
[ERROR]                    ^
```
We are using jflex-maven-plugin 1.7.0.

--- a/jflex/src/main/jflex/skeleton.nested
+++ b/jflex/src/main/jflex/skeleton.nested
@@ -197,7 +197,9 @@
   }
 
   /**
-   * Closes the input stream.
+   * Closes the input reader.
+   *
+   * @throws java.io.IOException if the reader could not be closed.
    */
   public final void yyclose() throws java.io.IOException {
     zzAtEOF = true;           // indicate end of file
@@ -297,6 +299,9 @@
     }
   }
 
+  /**
+   * Resets the input position.
+   */
   private final void yyResetPosition() {
       zzAtBOL  = true;
       zzAtEOF  = false;
@@ -312,6 +317,8 @@
 
   /**
    * Returns whether the scanner has reached the end of the reader it reads from.
+   *
+   * @return whether the scanner has reached EOF.
    */
   public final boolean yyatEOF() {
     return zzAtEOF;
@@ -320,6 +327,8 @@
 
   /**
    * Returns the current lexical state.
+   *
+   * @return the current lexical state.
    */
   public final int yystate() {
     return zzLexicalState;
@@ -338,6 +347,8 @@
 
   /**
    * Returns the text matched by the current regular expression.
+   *
+   * @return the matched text.
    */
   public final String yytext() {
     return new String( zzBuffer, zzStartRead, zzMarkedPos-zzStartRead );
@@ -361,7 +372,9 @@
 
 
   /**
-   * Returns the length of the matched text region.
+   * How many characters were matched.
+   *
+   * @return the length of the matched text region.
    */
   public final int yylength() {
     return zzMarkedPos-zzStartRead;

--- a/jflex/src/main/jflex/skeleton.nested-1.8.0
+++ b/jflex/src/main/jflex/skeleton.nested-1.8.0
@@ -197,7 +197,9 @@
   }
 
   /**
-   * Closes the input stream.
+   * Closes the input reader.
+   *
+   * @throws java.io.IOException if the reader could not be closed.
    */
   public final void yyclose() throws java.io.IOException {
     zzAtEOF = true;           // indicate end of file
@@ -297,6 +299,9 @@
     }
   }
 
+  /**
+   * Resets the input position.
+   */
   private final void yyResetPosition() {
       zzAtBOL  = true;
       zzAtEOF  = false;
@@ -312,6 +317,8 @@
 
   /**
    * Returns whether the scanner has reached the end of the reader it reads from.
+   *
+   * @return whether the scanner has reached EOF.
    */
   public final boolean yyatEOF() {
     return zzAtEOF;
@@ -320,6 +327,8 @@
 
   /**
    * Returns the current lexical state.
+   *
+   * @return the current lexical state.
    */
   public final int yystate() {
     return zzLexicalState;
@@ -338,6 +347,8 @@
 
   /**
    * Returns the text matched by the current regular expression.
+   *
+   * @return the matched text.
    */
   public final String yytext() {
     return new String( zzBuffer, zzStartRead, zzMarkedPos-zzStartRead );
@@ -361,7 +372,9 @@
 
 
   /**
-   * Returns the length of the matched text region.
+   * How many characters were matched.
+   *
+   * @return the length of the matched text region.
    */
   public final int yylength() {
     return zzMarkedPos-zzStartRead;

--- a/jflex/src/main/resources/jflex/skeleton.default
+++ b/jflex/src/main/resources/jflex/skeleton.default
@@ -153,7 +153,9 @@
 
 
   /**
-   * Closes the input stream.
+   * Closes the input reader.
+   *
+   * @throws java.io.IOException if the reader could not be closed.
    */
   public final void yyclose() throws java.io.IOException {
     zzAtEOF = true; // indicate end of file
@@ -187,7 +189,9 @@
     }
   }
 
-
+  /**
+   * Resets the input position.
+   */
   private final void yyResetPosition() {
       zzAtBOL  = true;
       zzAtEOF  = false;
@@ -204,6 +208,8 @@
 
   /**
    * Returns whether the scanner has reached the end of the reader it reads from.
+   *
+   * @return whether the scanner has reached EOF.
    */
   public final boolean yyatEOF() {
     return zzAtEOF;
@@ -212,6 +218,8 @@
 
   /**
    * Returns the current lexical state.
+   *
+   * @return the current lexical state.
    */
   public final int yystate() {
     return zzLexicalState;
@@ -230,6 +238,8 @@
 
   /**
    * Returns the text matched by the current regular expression.
+   *
+   * @return the matched text.
    */
   public final String yytext() {
     return new String(zzBuffer, zzStartRead, zzMarkedPos-zzStartRead);
@@ -251,7 +261,9 @@
 
 
   /**
-   * Returns the length of the matched text region.
+   * How many characters were matched.
+   *
+   * @return the length of the matched text region.
    */
   public final int yylength() {
     return zzMarkedPos-zzStartRead;

